test/tests/tracesConverterTest.js

```diff
@@ -338,10 +339,61 @@ describe('Traces converter tests', function () {
         };
         var trace = getRealtimeData(statement);
         should(trace).eql(resultTrace);
-        trace = getElasticSearchData(statement);
 
-        delete resultTrace.gameplayId;
-        delete resultTrace.versionId;
+        done();
+    });
+
+
+    it('should correctly convert result.extensions to ext object', function (done) {
+
+        var name = '57345599db69cf4200fa41d971088';
+        var event = 'skipped';
+        var target = 'testMenu';
+        var timestamp = '2016-05-16T11:48:25Z';
+        var type = 'testType';
+        var statement = {
+            id: '19de3bf2-6b7f-4399-a71b-da5f3674c8f8',
+            actor: {
+                name: name,
+                account: {
+                    homePage: 'http://a2:3000/',
+                    name: 'Anonymous'
+                }
+            },
+            verb: {
+                id: 'http://id.tincanapi.com/verb/' + event
+            },
+            object: {
+                id: 'http://example.com/games/SuperMarioBros/Screens/' + target,
+                definition: {
+                    extensions: definitionObj.extensions,
+                    type: 'https://rage.e-ucm.es/xapi/seriousgames/activities/' + type
+                }
+            },
+            result: {
+                extensions: {
+                    ext1: '123',
+                    '/ext2': '456',
```

Comment:
Add specific errors (describing the problem) to each parse function like getType, getTarget, getEvent...